quandl/model/dataset.py

```diff
@@ -29,6 +31,7 @@ def __init__(self, code, raw_data=None, **options):
         self.options = options
 
     def data(self, **options):
+        handle_not_found_error = options.pop('handle_not_found_error', False)
```

Comment:
https://github.com/quandl/quandl-python/blob/AP-1624/backwards-compatibility/quandl/model/dataset.py#L45 inserts blank entries into the pandas dataframe instead of erroring out early (something data team wanted to keep for the old `Quandl.get` flow